spec/features/users/admin_nav_spec.rb

```diff
@@ -37,16 +28,7 @@
   end
 
   it 'does not have access to cart or merchant' do
-    admin = User.create(
-      password: 'secure', 
-      name: 'Bob G', 
-      address: '123 Main St.', 
-      city: 'Denver', 
-      state: 'CO', 
-      zip: '80202', 
-      email: "bob@gmail.com", 
-      role: 3
-    )
+    admin = User.create(name: 'Bob', address: '123 Main', city: 'Denver', state: 'CO', zip: 80_233, email: 'bob@email.com', password: 'secure', role: 3)
```

Comment:
While this is not a big issue, having this all on one line is hard to read. We should move the arguments to new lines for readability.

Reply:
I see what you mean. I didn't even realize they were like that in master. I had done this on my end to make the tests pass. We can absolutely make them multiple lines if that is easier to read.